### ISSUE TITLE
Drag and Drop functionality with Nested Tree

### ISSUE THREAD

**Issue by bipin-sakariya** [corupta/react-native-nested-dnd#2]
Hello there,
Grateful for finding this featured external library plugin.

I have received this library reference from the below link,
https://stackoverflow.com/questions/59103114/how-to-implement-nested-drag-and-drop-in-react-native

I would like to update you that I am using the same library with the Drag and drop functionality. My feature requirements are related to the implementation of nested drag and drop with Tree. So I would like to ask you that as you have mentioned on your page, Did you find anything related to your ref link? Are there any updates from your end?
https://primetwig.github.io/react-nestable/dist/example/

Please let me know if there's any progress or if you are working with this.

Looking forward to hearing back from you.
Thanks.

**Comment by corupta** [comment]
Hi, there!

After the original stackoverflow post, I've forked & updated the underlying dnd library so that it supports choosing how many number of items, starting with the selected one should be dragged. Doing so, allowed me to remove my hacky method (quickly replacing items), and it works more smoothly, now.

With the same idea, for the tree case, if you move an item, it's moving count should be the number of items it has below (as children/nested children). That would allow dragging lists with nested children directly. So, instead of using this library, you can look at it as a reference and create a tree version using the underlying dnd library.

I only needed a package like I did at the time, so I was done with it, actually. But creating a tree version, also is a good idea. I would definitely want to do it, for fun. Yet, I'm very busy with other stuff for at least 2-3 weeks. 

To sum up, I cannot promise but I probably can create the tree version at the first week of November.
